firestore/FirestoreExample/RestaurantsTableViewController.swift

```diff
@@ -151,7 +152,7 @@ class RestaurantsTableViewController: UIViewController, UITableViewDataSource, U
     stopObserving()
   }
 
-  @IBAction func didTapPopulateButton(_ sender: Any) {
+  @IBAction func didTapPopulateButton(_ sender: Any) throws {
```

Comment:
`throws` is (was?) unsafe in `@IBAction` functions. See https://stackoverflow.com/a/30841323

Reply:
Oooh! I'm surprised the app worked with change in place.